fluent/client/ws/connection.go

```diff
@@ -235,6 +275,48 @@ type connMsg struct {
 	err     error
 }
 
+func (wsc *connection) runReadLoop(nextMsg chan connMsg) {
```

Comment:
The `if wsc.log` conditions raised the cognitive complexity and the linter rightly complained, so I moved the logic to a separate function. Otherwise no code change.